CsvParser/csvParser.go

```diff
@@ -172,7 +174,7 @@ func dynamicallyMakeDir(originPath string, locationCode string) string {
 	return originPath
 }
 
-//uses all the helper functions above to dynamically store incidents in a file tree hierachy
+// Uses all the helper functions above to dynamically store incidents in a file tree hierachy
```

Comment:
Remove the first part of this comment so that it is condensed to "Dynamically store incidents in a file tree hierarchy (typo)"